tests/Mixins/StoreCollectionTest.php

```diff
@@ -38,4 +39,26 @@ public function can_store_a_collection_as_excel_on_non_default_disk()
         $this->assertTrue($response);
         $this->assertFileExists(__DIR__ . '/../Data/Disks/Test/collection-store.xlsx');
     }
+
+    /**
+     * @test
+     */
+    public function can_store_a_collection_with_headings_as_excel()
+    {
+        $collection = new Collection([
+            ['column_1' => 'test', 'column_2' => 'test'],
+            ['column_1' => 'test', 'column_2' => 'test'],
+        ]);
+
+        $response = $collection->storeExcel('collection-headers-store.xlsx', null, Excel::XLSX, true);
+
+        $file = __DIR__ . '/../Data/Disks/Local/collection-headers-store.xlsx';
+
+        $this->assertTrue($response);
+        $this->assertFileExists($file);
+        if (is_file($file)) {
+            $array = $this->readAsArray($file, Excel::XLSX);
+            $this->assertEquals($collection->collapse()->keys()->all(), collect($array)->first());
```

Comment:
I would prefer an explicit key check, instead of using the same method as in the implementation. Would make it more explicit to understand what is being tested.